Assets/00_Content/Scripts/Menu/Lobby.cs

```diff
@@ -0,0 +1,56 @@
+using Player;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace Menu {
+
+    public class Lobby : MonoBehaviour
```

Comment:
Would it be fitting to place the lobby script on the canvas or an object in the canvas?